docker/README.md

```diff
@@ -0,0 +1,4 @@
+# build 
+sudo docker build -t seismic-deeplearning . 
+# run
+sudo docker run --rm -it -p 8888:8888 seismic-deeplearning 
```

Comment:
There's no -v option to mount the data which was downloaded?

Reply:
I'm working on it.. this is just a draft for now. :)